src/main/scala/cromwell/binding/types/WdlStringType.scala

```diff
@@ -12,6 +12,7 @@ case object WdlStringType extends WdlPrimitiveType {
     case s: String => WdlString(s)
     case s: JsString => WdlString(s.value)
     case s: WdlString => s
+    case f: WdlFile => WdlString(f.value)
```

Comment:
I don't believe the spec actually mentions anything about coercions, but it definitely should at some point.  I'll make a note of it for the next draft